bin/git-plop

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+find_plop_json() {
+  current_dir=$(pwd)
+
+  while [ "$current_dir" != "/" ]; do
+    if [ -f "$current_dir/.plop.json" ]; then
+      echo "$current_dir/.plop.json"
+      return 0
+    fi
+    current_dir=$(dirname "$current_dir")
+  done
+
+  echo "Error: .plop.json not found"
+  return 1
+}
+
+plop_json_path=$(find_plop_json)
+
+if [ "$plop_json_path" == "Error: .plop.json not found" ]; then
```

Comment:
to simplify you can just don't print anything and return 1 in find_plop_json, and in this if just check that variable is empty, and then print error message